core/src/test/resources/integrationTestScript.xml

```diff
@@ -129,6 +129,18 @@
                     <value>bulkload</value>
                     <value>--noprogress</value>
                 </list>
+                
+                <!-- data_structural_variants -->
+                <list>
+                    <value>org.mskcc.cbio.portal.scripts.ImportProfileData</value>
+                    <value>--data</value>
+                    <value>src/test/scripts/test_data/study_es_0/data_structural_variants.txt</value>
+                    <value>--meta</value>
+                    <value>src/test/scripts/test_data/study_es_0/meta_structural_variants.txt</value>
```

Comment:
perhaps add the genePanel test here...to study_es_0

Reply:
The meta file in `study_es_0` refers to a gene panel: 
```
gene_panel: example_gene_panel_stable_id
```